Changelog.md

```diff
@@ -13,6 +13,7 @@ Compiler Features:
  * SMTChecker: Support try/catch statements.
  * SMTChecker: Output internal and trusted external function calls in a counterexample's transaction trace.
  * SMTChecker: Synthesize untrusted functions called externally.
+ * Command Line Interface: Allow "=" as separator between library name and address in ``--libraries`` commandline option.
```

Comment:
Apparently we are also dropping support for non-0x prefix? If so, we should mention it here.

Reply:
OK. Added.